CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# Docker GoCD agent 17.12.0
+
+* [#46](https://github.com/gocd/docker-gocd-server/issues/46) - add support for custom entry-point scripts on startup. Any executables volume mounted in `/docker-entrypoint.d` will be executed before starting the GoCD server. This will allow users to perform initialization when running the container, or building images that derive from the official GoCD docker agent images.
+* Add ability to install custom plugins as part of bootstrap process
```

Comment:
Kill this line. THis feature is not implemented in the agent, since it makes no sense.